### ISSUE TITLE
Images Not Aligned In Two Column Layout

### ISSUE THREAD

**Issue by rjhilgefort** [gsuitedevs/md2googleslides#16]
When I try to have a slide with two columns and an image on the right side, I have to align that image when I generate my slides. Here's an example slide:

```markdown
# Foo

- Foo bullet
- Bar bullet
- Baz bullet

{.column}

![](https://upload.wikimedia.org/wikipedia/commons/thumb/2/2f/Google_2015_logo.svg/1200px-Google_2015_logo.svg.png)

```

Thanks for the tool! Really enjoy being able to create my decks and have them be available for PRs!

**Comment by epitre** [comment]
I have the same problem. It does not seem possible to have an image on the right side :/

**Comment by sqrrrl** [comment]
Yeah, I'll have to rework how images are handle. They're not currently associated with the text containers/shapes, which is what the columns.

Will think about how to handle this better.

**Comment by sqrrrl** [comment]
Fixed in master, will be in next release (0.5). Should get around to pushing that shortly.

**Comment by rjhilgefort** [comment]
Nice! Thank you @sqrrrl !!
